sstables/sstables.hh

```diff
@@ -498,6 +504,7 @@ public:
         future<> snapshot(const sstable& sst, sstring dir, absolute_path abs) const;
         future<> quarantine(const sstable& sst, delayed_commit_changes* delay);
         future<> activate(const sstable& sst, generation_type gen, delayed_commit_changes* delay);
+        future<> to_staging(const sstable& sst, generation_type gen);
```

Comment:
The other names are verbs, while this is just a direction.
Not sure how  verb version of this would sound like though.

Reply:
> Not sure how verb version of this would sound like though.

That's exactly the reason it's named like that. 
Would put_on_stage() be too dramatic?